gapis/resolve/resource_data.go

```diff
@@ -74,9 +74,10 @@ func buildResources(ctx context.Context, p *path.Command) (*ResolvedResources, e
 	for k, v := range resources {
 		res, err := v.ResourceData(ctx, state)
 		if err != nil {
-			return nil, fmt.Errorf("Could not get data for resource %v", k)
+			resourceData[k] = err;
```

Comment:
[Okay, you are making me feel uneasy now.](https://github.com/google/gapid/pull/212#discussion_r109881143)
I swear I didn't see this before making those comments...